src/NetAppFiles/NetAppFiles/Az.NetAppFiles.psd1

```diff
@@ -92,17 +92,17 @@ CmdletsToExport = 'Get-AzNetAppFilesAccount', 'New-AzNetAppFilesAccount',
                'Restore-AzNetAppFilesReplication', 
                'Resume-AzNetAppFilesReplication', 
                'Get-AzNetAppFilesVolumeReplication', 'Restore-AzNetAppFilesVolume', 
-               'Set-AzNetAppFilesVolumePool', 'New-AzNetAppFilesSnapshotPolicy', 
-               'Set-AzNetAppFilesSnapshotPolicy', 
+               'Set-AzNetAppFilesVolumePool', 'Unlock-AzNetAppFilesVolumeFileLock',
+               'New-AzNetAppFilesSnapshotPolicy',  'Set-AzNetAppFilesSnapshotPolicy', 
                'Get-AzNetAppFilesSnapshotPolicy', 
                'Update-AzNetAppFilesSnapshotPolicy', 
                'Remove-AzNetAppFilesSnapshotPolicy', 
                'New-AzNetAppFilesBackupPolicy', 'Get-AzNetAppFilesBackupPolicy', 
                'Set-AzNetAppFilesBackupPolicy', 'Update-AzNetAppFilesBackupPolicy', 
                'Remove-AzNetAppFilesBackupPolicy', 'New-AzNetAppFilesBackup', 
                'Get-AzNetAppFilesBackup', 'Update-AzNetAppFilesBackup', 
-               'Remove-AzNetAppFilesBackup', 'Get-AzNetAppFilesVolumeBackupStatus', 
-               'Get-AzNetAppFilesVolumeRestoreStatus', 'Get-AzNetAppFilesVault', 
```

Comment:
Looks you remove `Get-AzNetAppFilesVault`. You can also remove the source file `GetAzureRmNetAppFilesVault.cs`

Reply:
Removed